README.md

```diff
@@ -6,39 +6,39 @@
 
 ## উদ্দেশ্য
 
-ডেভেলপারদের জন্যে চিটশিট খুবই কাজের জিনিস। আমাদের পক্ষে অনেক কমান্ড বা কোনো লাইব্রেরীর সব ফাংশন একসাথে মনে রাখা সম্ভব হয় না। সেক্ষেত্রে আমরা যদি একটা লিস্ট করি যে অমুক কমান্ড বা ফাংশন অমুক কাজ করে তাহলে খুব সহজেই পরবর্তিতে কেউ কাজ করতে গেলে শুধুমাত্র এই লিস্টগুলো দেখলেই প্রয়োজনীয় সবকিছু খুঁজে পাবে। আমরা আমাদের পুরো নলেজটাকে বিভিন্ন বিভাগে ভাগ করে শুধুমাত্র আসল ফাংশানালিটিগুলো খুব ছোট্টো করে বর্ণনা দিয়ে লিস্ট করতে পারি। যেগুলোতে পরবর্তিতে কোনো ডেভেলপার জাস্ট একটু চোখ বুলিয়ে নিলেই তার কাঙ্ক্ষিত জিনিসটা খুঁজে পাবে। এরকম চিটশিট [এখানেও](https://devhints.io/) পাবেন। কিন্তু আমাদের লক্ষ্য হচ্ছে সম্পূর্ণ বাংলায় চিটশিট করা।
+ডেভেলপারদের জন্যে চিটশিট খুবই কাজের জিনিস। কারণ আমাদের পক্ষে অনেক কমান্ড বা কোনো লাইব্রেরীর সব ফাংশন একসাথে মনে রাখা সম্ভব হয় না। সেক্ষেত্রে আমরা যদি একটা তালিকা করি যে অমুক কমান্ড বা ফাংশন অমুক কাজ করে তাহলে খুব সহজেই পরবর্তিতে কেউ কাজ করতে গেলে শুধুমাত্র সেই তালিকা দেখলেই প্রয়োজনীয় সবকিছু খুঁজে পাবে। আমরা আমাদের পুরো জ্ঞানকে বিভিন্ন বিভাগে ভাগ করে শুধুমাত্র আসল ফাংশানালিটিগুলো খুব ছোট্টো করে বর্ণনা দিয়ে তালিকা তৈরি করতে পারি। যেগুলোতে পরবর্তিতে কোনো ডেভেলপার শুধু একটু চোখ বুলিয়ে নিলেই তার কাঙ্ক্ষিত জিনিসটা খুঁজে পাবে। এরকম চিটশিট [এখানেও](https://devhints.io/) পাবেন। কিন্তু আমাদের লক্ষ্য হচ্ছে সম্পূর্ণ বাংলায় চিটশিট তৈরি করা।
 
 ## কিভাবে কাজ করে
 
-এই প্রোজেক্ট পুরোপুরি গিটহাব বেইসড। এখানেই [**data**](https://github.com/devsonket/devsonket.github.io/tree/develop/data) নামক ডিরেক্টরির ভিতরে আমাদের সবগুলো চিটশীট **json** ফাইল আকারে আছে। আমাদের মেইন অ্যাপ্লিকেশন এখান থেকেই ডাটাগুলো নিয়ে শো করাবে। আমাদের বর্তমানে দুই ফরম্যাটের চিটশীট সাপোর্ট করছে।
+এই প্রকল্পটি পুরোপুরি গিটহাব ভিত্তিক। এখানেই [**data**](https://github.com/devsonket/devsonket.github.io/tree/develop/data) নামক ডিরেক্টরির ভিতরে আমাদের সবগুলো চিটশীট **json** ফাইল আকারে আছে। আমাদের মেইন অ্যাপ্লিকেশন এখান থেকেই ডাটাগুলো নিয়ে শো করাবে। আমাদের বর্তমানে দুই ফরম্যাটের চিটশীট সাপোর্ট করছে।
```

Comment:
প্রজেক্ট is better than প্রকল্প

Reply:
সংশোধন করা রয়েছে। কমিট– ad2e3d470bf1e5be28ff11cc5c8a6c0bb46b3d4f